README.md

```diff
@@ -34,6 +34,16 @@ This will start hops in development mode. Visit [http://localhost:8080](http://l
 
 If you want to build the JS bundles for production, run `npm run build` in your terminal and to start a Node.js server just type in `npm start --production`.
 
+### Project Templates
+
+Hops comes with a couple of different project templates you can use with `hops init --template`. If you do not explicitly select one, `hops init` defaults to `hops-template-react`.
```

Comment:
Should we provide a complete example command?
As in: `hops init --template hops-template-graphql`?